sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -25,6 +25,10 @@
 
 #include "usm_allocator.hpp"
 
+#ifdef _WIN32
+#define setenv(name, value, overwrite) _putenv_s(name, value)
```

Comment:
Let's not use macros without a good reason. Please just create a new function that works for both Linux and Windows.